mage/query-modules/python/temporal-graph-networks.md

```diff
@@ -0,0 +1,415 @@
+---
+id: temporal-graph-networks
+title: temporal_graph_networks
+sidebar_label: temporal_graph_networks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const Highlight = ({children, color}) => (
+  <span
+    style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+    {children}
+  </span>
+);
+
+[![docs-source](https://img.shields.io/badge/source-node2vec_online-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/blob/main/python/tgn.py)
+
+## Abstract
+
+The **temporal_graph_networks (TGNs)** is a **[graph neural network (GNN)](https://distill.pub/2021/gnn-intro/)** method on dynamic graphs. 
```

Comment:
Consider rewriting this; I've left a comment at the other place this sentence is found